tests/test_other.py

```diff
@@ -788,9 +788,11 @@ def test_cxx11(self):
 
   # Regression test for issue #4522: Incorrect CC vs CXX detection
   def test_incorrect_c_detection(self):
+    # This auto-detection only works for the compile phase.
+    # For linking you need to use `em++` or pass `-x c++`
```

Comment:
Because if you use `emcc` you don't get libc++ or libc++abi included in your link.. and you will have undefined symbols.  

I could make that a separate change perhaps?

Reply:
Oh I see. No I think this is fine as is.